src/prelude.rs

```diff
@@ -15,11 +15,12 @@ pub use list_store::ListStoreExtManual;
 pub use output_stream::OutputStreamExtManual;
 pub use pollable_input_stream::PollableInputStreamExtManual;
 pub use pollable_output_stream::PollableOutputStreamExtManual;
+pub use settings::SettingsExtManual;
 pub use socket::*;
 pub use socket_listener::SocketListenerExtManual;
 #[cfg(any(unix, feature = "dox"))]
 pub use unix_input_stream::UnixInputStreamExtManual;
 #[cfg(any(unix, feature = "dox"))]
 pub use unix_output_stream::UnixOutputStreamExtManual;
 #[cfg(any(unix, feature = "dox"))]
-pub use unix_socket_address::{UnixSocketAddressExtManual, UnixSocketAddressPath};
+pub use unix_socket_address::{UnixSocketAddressExtManual, UnixSocketAddressPath};
```

Comment:
This change cause CI fail on regen_check step

Reply:
Yes, it needs a `cargo fmt` run

Reply:
Ah, normally CLion runs that for me, but it's a per-project setting I forgot to check here